demo/radio-button-basic-demos.html

```diff
@@ -5,22 +5,93 @@
         display: block;
       }
     </style>
+    <h3>Basic usage</h3>
+    <vaadin-demo-snippet id="basic-usage">
+      <template preserve-content>
+        <vaadin-radio-group label="Label" theme="vertical">
```

Comment:
Why do we yse `theme="vertical"` from almost all the examples now?
I think it should be only used in "Theme Variants".

Reply:
That is what we discussed with Jouni. He said that it's better to have the demos in a vertical position.

Reply:
If that was discussed with Jouni, then let's keep it.